README.md

```diff
@@ -1827,6 +1828,7 @@ Update Time, 5 active automations, webhooks.
 
 ## Other Free Resources
 
+  * [Buff.tools](https://buff.tools/) - Discover a diverse range of everyday tools, from SEO and image optimization to developer utilities and website management aids, offering comprehensive solutions for your digital needs for Free.
```

Comment:
We lost entries only once so please find the best place to add it 

thanks

Reply:
sorry about that. Will refactor it.